addons/zeus/stringtable.xml

```diff
@@ -253,7 +253,7 @@
                 <German>Anzahl der zu hinzuzufügenden Raketen</German>
             </Key>
             <Key ID="STR_GRAD_ZEUS_AmountHandgranades">
-                <English>Number of grenades to add</English>
```

Comment:
himmelherrgottnochmal. 

* amount  = unzählbare Dinge (die man dann mit Einheit, zB Liter oder Kilogramm ausrückt) 
* number, count =  zählbare Dinge (eins zwei drei vier ohne Einheit)

also solang du die Granaten nicht in Zentnern bestellst, ist "amount" falsch.

Reply:
So viele wie wir nutzen werden die im Zentner bewegt 🗡️